concent_api/core/tests/test_core_views.py

```diff
@@ -286,6 +286,40 @@ def test_send_reject_message_save_as_receive_out_of_band_status(self):
         self.assertEqual(reject_response.status_code, 202)
         self.assertEqual(Message.objects.last().type, message.RejectReportComputedTask.TYPE)
 
+    @freeze_time("2017-11-17 10:00:00")
```

Comment:
Please use `with freeze_time()` around the tested call instead of wrapping the whole function.

Reply:
:+1: